std-bits/table/src/main/java/org/enso/table/data/column/operation/map/numeric/LongRoundOp.java

```diff
@@ -0,0 +1,81 @@
+package org.enso.table.data.column.operation.map.numeric;
+
+import org.enso.polyglot.common_utils.Core_Math_Utils;
+import org.enso.table.data.column.operation.map.TernaryMapOperation;
+import org.enso.table.data.column.operation.map.MapOperationProblemBuilder;
+import org.enso.table.data.column.storage.numeric.AbstractLongStorage;
+import org.enso.table.data.column.storage.numeric.LongStorage;
+import org.enso.table.data.column.storage.Storage;
+import org.enso.table.error.UnexpectedTypeException;
+import org.graalvm.polyglot.Context;
+
+import java.util.BitSet;
+
+/** An operation rounding integers. */
+public class LongRoundOp extends TernaryMapOperation<Long, AbstractLongStorage> {
+
+    /** Minimum value for the `n` parameter to `roundLong`. */
+    private static final long ROUND_MIN_LONG = -99999999999999L;
+
+    /** Minimum value for the `n` parameter to `roundLong`. */
+    private static final long ROUND_MAX_LONG = 99999999999999L;
+
+    public LongRoundOp(String name) {
+        super(name);
+    }
+
+    @Override
+    public Storage<Long> runTernaryMap(AbstractLongStorage storage, Object decimalPlacesObject, Object useBankersObject, MapOperationProblemBuilder problemBuilder) {
+        if (decimalPlacesObject == null) {
+            throw new IllegalArgumentException("decimalPlacesObject must not be null");
```

Comment:
Out of curiosity: shall not the exception be `NullPointerException`? Or use `java.util.Objects.requireNonNull`...